provider/maas/config_test.go

```diff
@@ -15,7 +14,7 @@ import (
 
 // Ensure MAAS provider supports the expected interfaces.
 var (
-	_ config.ConfigSchemaSource = (*maasEnvironProvider)(nil)
+	_ config.ConfigSchemaSource = (*MaasEnvironProvider)(nil)
```

Comment:
I've been getting rid of these. If we wanted to do that check we'd do it in the test code (if at all).